### PR TITLE
Model: CNN

### PR DESCRIPTION
**Pull request by buesst1**

### CODE REVIEW

**Comment by ghost** [comment]
# RUN: CNN, 2023-06-09 11-17-59

## Hyperparameters

| Path        | Param                                  | main                           | workspace   |
|-------------|----------------------------------------|--------------------------------|-------------|
| params.yaml | dvclive.data.batch_size                | -                              | 64          |
| params.yaml | dvclive.data.num_workers               | -                              | 0           |
| params.yaml | dvclive.data.output_size               | -                              | 6           |
| params.yaml | dvclive.data.params.columns            | numeric                        | -           |
| params.yaml | dvclive.data.train_val_split           | -                              | 0.8         |
| params.yaml | dvclive.data.type                      | Sklearn_Segments               | NDArray     |
| params.yaml | dvclive.model                          | HistGradientBoostingClassifier | CNN         |
| params.yaml | dvclive.model_hparams.learning_rate    | 0.2                            | 0.05        |
| params.yaml | dvclive.model_hparams.max_bins         | 255                            | -           |
| params.yaml | dvclive.model_hparams.max_iter         | 2000                           | -           |
| params.yaml | dvclive.model_hparams.max_leaf_nodes   | 31                             | -           |
| params.yaml | dvclive.model_hparams.min_samples_leaf | 20                             | -           |
| params.yaml | dvclive.model_hparams.momentum         | -                              | 0.9         |
| params.yaml | dvclive.model_hparams.num_epochs       | -                              | 15          |
| params.yaml | dvclive.model_hparams.weight_decay     | -                              | 0.0005      |
| params.yaml | dvclive.seed                           | 42                             | 40          |

## Metrics

| Path                 | Metric           | main    | workspace   | Change   |
|----------------------|------------------|---------|-------------|----------|
| dvclive/metrics.json | epoch            | -       | 15          | -        |
| dvclive/metrics.json | step             | -       | 1440        | -        |
| dvclive/metrics.json | test.epoch.acc   | 0.98289 | 0.93632     | -0.04657 |
| dvclive/metrics.json | test.epoch.f1    | 0.98085 | 0.93133     | -0.04952 |
| dvclive/metrics.json | test.step.acc    | -       | 0.93632     | -        |
| dvclive/metrics.json | test.step.f1     | -       | 0.93133     | -        |
| dvclive/metrics.json | train.epoch.acc  | 1.0     | 0.92755     | -0.07245 |
| dvclive/metrics.json | train.epoch.f1   | 1.0     | 0.91827     | -0.08173 |
| dvclive/metrics.json | train.epoch.loss | -       | 0.18744     | -        |
| dvclive/metrics.json | train.step.acc   | -       | 1.0         | -        |
| dvclive/metrics.json | train.step.f1    | -       | 1.0         | -        |
| dvclive/metrics.json | train.step.loss  | -       | 0.08281     | -        |
| dvclive/metrics.json | val.epoch.acc    | -       | 0.93377     | -        |
| dvclive/metrics.json | val.epoch.f1     | -       | 0.93071     | -        |
| dvclive/metrics.json | val.epoch.loss   | -       | 0.18451     | -        |
| dvclive/metrics.json | val.step.acc     | -       | 0.93377     | -        |
| dvclive/metrics.json | val.step.f1      | -       | 0.93071     | -        |
| dvclive/metrics.json | val.step.loss    | -       | 0.18451     | -        |

## Plots

### Confusion Matrix

Confusion Matrix

### CV Metrics

CV Metrics

**Comment by gabrieltorresgamez** [comment]
Don't merge!!!
